db/agents.json

```diff
@@ -1714,6 +1714,24 @@
         "Spotify/4.3.2.0 Polestar/30 (Polestar)"
       ]
     },
+    {
+      "regex": "^SpreakerPodcast/.+(Android",
```

Comment:
Thanks for the PR @nonno!

Realizing that the README isn't clear... but we actually use `agents.yml` as the source-of-truth, and generate the other files from that.

Do you mind moving these over to agents.yml, and just editing that 1 file?  Then I can re-lock everything.  (The html file is also generated from the lock process).